src/stores/OverlayStore/OverlayStore.ts

```diff
@@ -1208,7 +1208,7 @@ export class OverlayStore {
     }
 
     @action minSize(frame?: FrameStore) {
```

Comment:
Let's remove the optional `?` because `frame` is always provided and is required in the calculation.